EPAS 15 new release branch (#2283)
EnterpriseDB/docs: product_docs/docs/epas/15/epas_compat_table_partitioning/04_partitioning_commands_compatible_with_oracle_databases/01_create_table_partition_by/index.mdx

Use the `SUBPARTITION TEMPLATE` with a list, range, or hash subpartitioning to create default subpartitions for each table partition for which you don't explicitly define subpartitions. For more information about setting subpartition template, see [ALTER TABLE...SET SUBPARTITION TEMPLATE](../../04_partitioning_commands_compatible_with_oracle_databases/11_alter_table_set_subpartition_template/#alter_table_set_subpartition_template).

> Use the `ENABLE ROW MOVEMENT` with a list, range, or hash partition tables to support the migration of tables with similar syntax from oracle databases, however it do not enable the actual row movement. Also, this syntax is not supported with `ALTER TABLE` command.

Reviewer: ```suggestion
Use the `ENABLE ROW MOVEMENT` with a list, range, or hash partition table to support the migration of tables with similar syntax from Oracle databases; however, it doesn't enable the actual row movement. This syntax is not supported with the `ALTER TABLE` command.
```

Reviewer 2: Use `ENABLE ROW MOVEMENT` with a list, range, or hash partition table to support migrating tables with similar syntax from Oracle databases. However, it doesn't enable the actual row movement. This syntax isn't supported with the `ALTER TABLE` command.
